sota-launcher/examples/config.toml

```diff
@@ -0,0 +1,2 @@
+environment = "ci"
+play_session = "PLAY_SESSION=1234567890abcdef1234567890abcdef12345678-id_token=1234567890abcdefghijklmnopqrstuvwxyz.1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABC.1234567890abcdefghijklmnopqrstuvwxyzABCDEFG&access_token=abcdefghijklmnop&auth_plus_access_token=1234567890abcdefghijklmnopqrstuvwxy.1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890abcdefghijklmn.1234567890abcdefghijklmnopqrstuvwxyzABCDEFG&csrfToken=1234567890abcdef1234567890abcdef12345678-1234567890123-1234567890abcdef12345678&namespace=auth0%7C1234567890abcdefghijklmn"
```

Comment:
Because our web API expects it. Extracting it every hour or so really sucks and I'd be happy to get rid of it entirely if there's another way.